.github/workflows/deploy-stage.yml

```diff
@@ -26,3 +28,21 @@ jobs:
           cf_org: hhs-acf-ohs-hses
           cf_space: ct-stage
           push_arguments: "--vars-file config/deployment/stage.yml --var rails_master_key=$RAILS_MASTER_KEY --var hses_auth_client_secret=$HSES_AUTH_CLIENT_SECRET"
+
+      - name: Check for changes to Terraform
+        id: changed-files
+        uses: tj-actions/changed-files@v1.1.2
+        with:
+          files: |
+            terraform/shared/**
```

Comment:
[comment] I am not actually sure if this syntax will work, as I didn't see a lot of guidance about directories in the [changed-files documentation](https://github.com/tj-actions/changed-files), but I hoping it will?

Reply:
[suggestion] I don't think it will work, but just `terraform/shared` and `terraform/stage` will.

Basically what it's doing with these is running `git diff --name-only | grep -E FILE_NAME` to filter the files changed down to the name. I did a quick and dirty test and a file with `**` at the end had this error: `grep: repetition-operator operand invalid`